src/com/maddyhome/idea/vim/group/ChangeGroup.java

```diff
@@ -1751,7 +1751,7 @@ private boolean deleteText(final @NotNull Editor editor,
       if (type != null) {
         final int start = range.getStartOffset();
         VimPlugin.getMark().setMark(editor, MarkGroup.MARK_CHANGE_POS, start);
-        VimPlugin.getMark().setChangeMarks(editor, new TextRange(start, start));
+        VimPlugin.getMark().setChangeMarks(editor, new TextRange(start, start+1));
```

Comment:
I think we should add some tests to prove all the +1/-1 changes. Or does IdeaVim already have enough tests for marks?